packages/discovery/package.json

```diff
@@ -19,12 +19,16 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@inquirer/checkbox": "^1.5.0",
+    "@inquirer/select": "^1.3.1",
     "@l2beat/backend-tools": "^0.5.1",
     "@l2beat/discovery-types": "^0.8.0",
     "chalk": "^4.1.2",
+    "comment-json": "^4.2.3",
```

Comment:
We already import `jsonc-parser`, either don't use `comment-json` or replace `jsonc-parser` with it